### DISCUSSION TITLE
[UX] Delete recipes should be in the three dots menu

### DISCUSSION THREAD

**Discussion by simgunz** [mealie-recipes/mealie#3984] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

To delete a recipe one has to first click the Edit button and then the delete button. On most applications an object is deleted from the three dots menu, without the need to first edit it.

### Please provide a concise description of the feature that would resolve your issue.

 I think that moving the Delete button to the three dots menu and removing it from the Edit page will improve the usability.

I would place it at the end of this list:
![Screenshot_20240804_161209](https://github.com/user-attachments/assets/898e113d-14ae-42f3-9e27-a0183548188c)

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by johnpc** [comment]
It should also be available in the list view. There should be a confirmation "are you sure" dialog when clicked too.

**Reply by johnpc** [reply]
For what it's worth, I've PR'd this idea here: https://github.com/mealie-recipes/mealie/pull/4842

We'll see what the maintainers say.

**Comment by alanbernstein** [comment]
I'd like to note that deleting recipes is an action perhaps biased slightly toward new users, as they try out an unfamiliar interface and get unexpected import results. It took me 5 minutes to find the delete button, and I did not find it thanks to documentation or forum posts, but rather an LLM answer. I expected to be able to delete in the list view.

**Comment by ChipSkylark37** [comment]
Something I have been thinking of is an interactive 'demo'. 

I recently upgraded my Hubitat Hub (Home Automation) and on the new interface they had, they integrated something that walked you through performing actions

"Let's try creating a recipe, first, click over here" and it would follow you through the process / pages, showing you how to add a device, rename, delete, etc. 

Though this wouldn't be easy to implement by any means, it's something to consider since anyone new to a platform may get frustrated if they don't find the things they want to do easily.

Heck, a lot of people don't know you can link ingredients to steps, or create sections for ingredients / steps, etc. There's a lot that's there that is not _in your face_ or sometimes you just need a reminder about a cool feature you may not use that often.
